### PR TITLE
Library decisions go elsewhere

### PR DESCRIPTION
**Pull request by mike-burns**
Keep the guides as just that: guides. Our best practices are not the
place to keep decisions about which libraries are best at solving which
problem; that goes in templates and project generators.

Move oj into Suspenders in
https://github.com/thoughtbot/suspenders/pull/923 .

Inspired by #509.

### CODE REVIEW

**Comment by composerinteralia** [comment]
Makes sense, and seems to match https://github.com/thoughtbot/guides/pull/518

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,3 +8,14 @@ open a new pull request.
 Since these are our guides, we want everyone at thoughtbot to see them. We have
 a lot of offices across a lot of timezones, so we leave all PRs open for at
 least a week to get feedback from everyone.
+
+## Content
+
+Our goal is to set higher-level decisions instead of specific choices, to lead
```
[the remark this answers is not in the document]
Comment (mike-burns) [reply CONTRIBUTING.md]: Yeah true. Maybe I'll just kill the CONTRIBUTING change here and this entire commit is just removing oj.

**Comment by cpytel** [comment]
@mike-burns will you have a chance to revisit this soon?

**Comment by mike-burns** [comment]
@cpytel yes, this week.

(Friday was an office outing for NYC.)

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mike-burns) [reply CONTRIBUTING.md]: Or I'll be more specific. How's this re-write?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -8,3 +8,10 @@ open a new pull request.
 Since these are our guides, we want everyone at thoughtbot to see them. We have
 a lot of offices across a lot of timezones, so we leave all PRs open for at
 least a week to get feedback from everyone.
+
+## Content
+
+Decisions aboiut which libraries to use should live in template projects such
```
Comment (tysongach) [inline CONTRIBUTING.md]: `aboiut` → `about`

**Pull request merged**